recipes/vc/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: vc
+  version: 9   # [win and py27]
+  version: 10  # [win and py34]
+  version: 14  # [win and py35]
+
+build:
+  number: 0
+  track_features:
+    - vc9      # [win and py27]
+    - vc10     # [win and py34]
+    - vc14     # [win and py35]
+  skip: true   # [unix]
+
+requirements:
+  build:
+    - python
+
+  run:
+    - vs2008_runtime  # [win and py27]
+    - vs2010_runtime  # [win and py34]
+    - vs2015_runtime  # [win and py35]
```

Comment:
Usually you want to depend on a meta package that tracks one feature. But this one is kinda a meta-package that tracks 3 features. I think we also can add 3 meta packages each for a VC feature. But it also works like this with the versioning hack we did.